app_dart/lib/src/request_handlers/deflake_flaky_builders.dart

```diff
@@ -56,7 +56,8 @@ class DeflakeFlakyBuilders extends ApiRequestHandler<Body> {
     for (final _BuilderInfo info in eligibleBuilders) {
       final List<BuilderRecord> builderRecords =
           await bigquery.listRecentBuildRecordsForBuilder(kBigQueryProjectId, builder: info.name, limit: kRecordNumber);
-      if (builderRecords.length >= kRecordNumber && builderRecords.every((BuilderRecord record) => !record.isFlaky)) {
+      if (builderRecords.length >= kRecordNumber &&
```

Comment:
This is difficult to follow. Could you move it into a helper function that gives more context of what it's looking for?

Reply:
Done!